<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="35878" started-at="2017-09-05T07:15:58Z" finished-at="2017-09-05T07:16:34Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="35878" started-at="2017-09-05T07:15:58Z" finished-at="2017-09-05T07:16:34Z">
      <class name="testcases.account_center.test_case_022_account_center_msg_search">
        <test-method status="PASS" signature="setUp()[pri:0, instance:testcases.account_center.test_case_022_account_center_msg_search@7cdd422e]" name="setUp" is-config="true" duration-ms="4458" started-at="2017-09-05T15:15:58Z" finished-at="2017-09-05T15:16:03Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="test_account_center_msg_search()[pri:0, instance:testcases.account_center.test_case_022_account_center_msg_search@7cdd422e]" name="test_account_center_msg_search" duration-ms="30992" started-at="2017-09-05T15:16:03Z" finished-at="2017-09-05T15:16:34Z">
          <exception class="java.lang.ArrayIndexOutOfBoundsException">
            <message>
              <![CDATA[-3]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.ArrayIndexOutOfBoundsException: -3
	at java.util.ArrayList.elementData(ArrayList.java:400)
	at java.util.ArrayList.get(ArrayList.java:413)
	at pages.base.base_page.third_last_pageNum1(base_page.java:88)
	at pages.base.base_page.page_number(base_page.java:64)
	at testcases.account_center.test_case_022_account_center_msg_search.test_account_center_msg_search(test_case_022_account_center_msg_search.java:30)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.TestNG.privateMain(TestNG.java:1355)
	at org.testng.TestNG.main(TestNG.java:1324)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.ArrayIndexOutOfBoundsException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_account_center_msg_search -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:testcases.account_center.test_case_022_account_center_msg_search@7cdd422e]" name="tearDown" is-config="true" duration-ms="420" started-at="2017-09-05T15:16:34Z" finished-at="2017-09-05T15:16:34Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- testcases.account_center.test_case_022_account_center_msg_search -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
